supervisor/misc/tasks.py

```diff
@@ -140,16 +139,16 @@ async def _update_addons(self):
                 continue
 
             if addon.test_update_schema():
-                tasks.append(addon.update())
+                _LOGGER.info("Add-on auto update process %s", addon.slug)
+                try:
```

Comment:
Since we're doing an `await` in the for-loop, better to reverse the if-statement.

```python
if not addon.test_update_schema():
    _LOGGER.warning(…)
    continue

try:
    …
```